src/components/homepage/pokemonPictureCards/PokemonPictureCard.tsx

```diff
@@ -3,23 +3,67 @@ import { Link } from "react-router-dom";
 import styled from "styled-components";
 import ContainerPrototype from "../../prototypes/ContainerPrototype";
 import PokemonTypesElement from "./PokemonTypesElement";
-
 import getPokemonData from "../../../functions/api/getPokemonData";
 import capitalizeWords from "../../../functions/utilities/capitalizeWords";
 import { PokemonNumberPropsInterface, TypesColorsInt } from "../../../interfaces/miscInterfaces";
-import PokemonInterface from "../../../interfaces/pokemonInterface";
+import PokemonInterface, { Type } from "../../../interfaces/pokemonInterface";
 import typesColors from "../../../objects/typesColors";
 
+export default function PokemonPictureCard(props: PokemonNumberPropsInterface): React.ReactElement {
+    const [pokemonInfo, setPokemonInfo] = useState<PokemonInterface>();
+
+    async function getData(pokemonNumber: number): Promise<void> {
+        try {
+            const data: PokemonInterface = await getPokemonData(pokemonNumber);
+            setPokemonInfo(data);
+        } catch (err) {
+            console.log(err);
+            return;
+        }
+    }
+
+    useEffect(() => {
+        getData(props.id);
+    }, [props.id]);
+
+    const renderPokemonTypes = (typesArray: Type[]): React.ReactElement[] =>
+        typesArray.reverse().map((x: Type) => <PokemonTypesElement typeName={capitalizeWords(x.type.name)} />);
```

Comment:
There is not unique key for What you are mapping. You must always add unique keys to react components to avoid rendering problems